e2e/pages/adf/process_services/processListDemoPage.ts

```diff
@@ -0,0 +1,153 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Util } from '../../../util/util';
+import { ContentListPage } from '../dialog/contentListPage';
+import { element, by, protractor } from 'protractor';
+
+export class ProcessListDemoPage {
+
+    appIdInput = element(by.css('input[data-automation-id="app-id"]'));
+    resetButton = element(by.cssContainingText('button span', 'Reset'));
+    emptyProcessContent = element(by.css('div[class="adf-empty-content"]'));
+    processDefinitionInput = element(by.css('input[data-automation-id="process-definition-id"]'));
+    processInstanceInput = element(by.css('input[data-automation-id="process-instance-id"]'));
+    stateSelector = element(by.css('mat-select[data-automation-id="state"'));
+    sortSelector = element(by.css('mat-select[data-automation-id="sort"'));
+
+    contentList = new ContentListPage();
+
+    async getAppDefinitionId(alfrescoJsApi, appModelId) {
+        let appDefinitions = await alfrescoJsApi.activiti.appsApi.getAppDefinitions();
+        let appDefinitionId = -1;
+
+        appDefinitions.data.forEach((appDefinition) => {
+            if (appDefinition.modelId === appModelId) {
+                appDefinitionId = appDefinition.id;
+            }
+        });
+
+        return appDefinitionId;
+    }
+
+    async getProcessTaskId(alfrescoJsApi, processId) {
+        let taskList = await alfrescoJsApi.activiti.taskApi.listTasks();
+        let taskId = -1;
+
+        taskList.data.forEach((task) => {
+            if (task.processInstanceId === processId) {
+                taskId = task.id;
+            }
+        });
+
+        return taskId;
+    }
+
+    getDisplayedProcessesNames() {
+        let nameColumn = by.css('div[class*="adf-datatable-body"] div[class*="adf-datatable-row"] div[title="Name"] span');
+        return this.contentList.getAllRowsColumnValues(nameColumn);
+    }
+
+    selectSorting(sort) {
+        Util.waitUntilElementIsVisible(this.stateSelector);
+        this.sortSelector.click();
+        let sortLocator = element(by.cssContainingText('mat-option span', sort));
+        Util.waitUntilElementIsVisible(sortLocator);
+        sortLocator.click();
+        return this;
+    }
+
+    selectStateFilter(state) {
+        Util.waitUntilElementIsVisible(this.stateSelector);
+        this.stateSelector.click();
+        let stateLocator = element(by.cssContainingText('mat-option span', state));
+        Util.waitUntilElementIsVisible(stateLocator);
+        stateLocator.click();
+        return this;
+    }
+
+    addAppId(appId) {
```

Comment:
On taskListDemoPage worked without lines 85,86. Here is not working with the same approach?